preprocessors/openstreetmap/osm_service.py

```diff
@@ -122,17 +123,14 @@ def process_streets_data(OSM_data, bbox_coordinates):
                     "lat": float(node.lat),
                     "lon": float(node.lon),
                 }
+                # Include tags for a street node if available
+                node_object.update(node.tags)
```

Comment:
While POIs under the ```amenity``` and ```building``` keys are off-route landmarks, this specifically ensures that attributes are extracted for nodes of a way if available and this is where we can get information on the traffic, crossing, etc.